src/main/java/com/quirko/app/GameController.java

```diff
@@ -6,41 +6,65 @@
 import com.quirko.logic.events.InputEventListener;
 import com.quirko.logic.events.MoveEvent;
 
+
 public class GameController implements InputEventListener {
 
-    private Board board = new SimpleBoard(25, 10);
+    private Level level = new Level(70);

```

Comment:
70 nedir? Bu değeri ismi manalı bir constanta taşı

Reply:
Integer is now being hold by a meaningful constant.